app/pages/SchoolPage.js

```diff
@@ -60,7 +62,7 @@ class SchoolPage extends React.Component {
           <SchoolTitle name={this.state.mainName} />
```

Comment:
Maybe the official_name could be optional and if it's missing then the school title renders nothing or just an empty div or something.